ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -62,6 +70,21 @@ const ListingCard = (props: ListingCardProps) => {
             </>
           )}
         </div>
+        {profile ? (
+          preferences?.favorites?.includes(props.listingID) ? (
+            <button onClick={() => props.removeFavorite(props.listingID)}>
+              <Icon symbol={"favorite"} size={"large"} />
+              {" Remove from Favorites"}
```

Comment:
Can you add this string to the JSON translations? Similarly for the `Add to Favorites` and `Sign in For Favs` below.

Reply:
updated